packages/client/src/components/MainNavigation.tsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
-import { Box, List, ListItem, Button, ButtonProps } from '@chakra-ui/react';
+import { List, ListItem, Button, ButtonProps, Flex } from '@chakra-ui/react';
 import {
   CollecticonFolder,
   CollecticonPlusSmall
 } from '@devseed-ui/collecticons-chakra';
 
 import SmartLink, { SmartLinkProps } from './SmartLink';
+import { UserInfo } from './auth/UserInfo';
+import { useAuth0 } from '@auth0/auth0-react';
```

Comment:
I see we are literally implementing this auth functionality against Auth0, using an Auth0-specific library.

I was hoping that, if at all, we could use Auth0 as a sample provider that we could then soon replace with KeyCloak in our projects, since all use the same flow (Authorization Code Flow with PKCE).

Is that the case?

Reply:
If it is indeed the case that this implementation is provider-agnostic, then we should rename the env vars and names inside the app to something not Auth0-specific, just "auth" or so.